src/carma-1-tenth/calc_route_deviation.py

```diff
@@ -0,0 +1,123 @@
+import sys
+import rosbag2_py
+import numpy as np
+import yaml
+import tqdm
+from rosidl_runtime_py.utilities import get_message
+from rclpy.serialization import deserialize_message
+import matplotlib.pyplot as plt
+import datetime as dt
+from scipy.interpolate import make_interp_spline
+import os
+
+STEERING_TO_SERVO_OFFSET = 0.425
+STEERING_TO_SERVO_GAIN = -0.55
+
+def get_rosbag_options(path, serialization_format="cdr", storage_id="sqlite3"):
```

Comment:
using db3 rosbag, will likely want to transition to mcap but likely out of scope

Reply:
the script grabs the storage_id from the rosbags metadata.yaml, so this should work for mcaps as well